CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Trino
```

Comment:
Trino → Trino Gateway

Reply:
Well .. I think it makes sense to keep it Trino.. since they contribute to the Trino Software Foundation and the trinodb org on GitHub .. but I am happy to change as well.. wdyt @dain

Reply:
This project isn't "Trino". See https://github.com/trinodb/trino-python-client/blob/master/.github/CONTRIBUTING.md

Reply:
Also the CLA is for the TSF

Reply:
I can see both sides ... hence asking @dain to chime in.

Reply:
I got an even better fix now .. the name can just be removed (and is therefore implied).